polkadot-parachains/statemine/src/constants.rs

```diff
@@ -14,19 +14,20 @@
 // limitations under the License.
 
 pub mod currency {
+	use kusama_runtime_constants as constants;
 	use node_primitives::Balance;
 
 	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9020).
-	pub const EXISTENTIAL_DEPOSIT: Balance = CENTS / 10;
+	pub const EXISTENTIAL_DEPOSIT: Balance = constants::currency::EXISTENTIAL_DEPOSIT / 10;
```

Comment:
The `EXISTENTIAL_DEPOSIT` is not directly equal to `constants::currency::EXISTENTIAL_DEPOSIT`, but a tenth of it? How does this make any sense, especially given that the previous code has `CENTS / 10`?

Reply:
If you check the [existential deposit in Kusama](https://github.com/paritytech/polkadot/blob/12ffb5683b05f9fe7ae0b4c0cd1c329e2e125942/runtime/kusama/constants/src/lib.rs#L23) it is `1 CENT`. As the comment above the code explains, we want the ED on Statemine to be 1/10th of the relay chain deposit, so that's why it originally was hard-coded to `CENTS / 10` and now uses the `kusama_runtime_constants` divided by 10.

The reason for wanting it to be 1/10th is that storage on the relay is more expensive to the overall network than on a parachain. (Which is part of the motivation for introducing Statemint/e as an assets parachain.)